networkx/algorithms/tests/test_dag.py

```diff
@@ -281,17 +281,29 @@ def test_descendants(self):
 
     def test_transitive_closure(self):
         G = nx.DiGraph([(1, 2), (2, 3), (3, 4)])
-        solution = [(1, 2), (1, 3), (1, 4), (2, 3), (2, 4), (3, 4)]
-        assert edges_equal(nx.transitive_closure(G).edges(), solution)
+        res = [(1, 2), (1, 3), (1, 4), (2, 3), (2, 4), (3, 4)]
+        assert edges_equal(nx.transitive_closure(G).edges(), res)
+
```

Comment:
Are these tests any different from before other than changing the variable name from `solution` to `res`?
If they are just that, let's leave them as `solution`. Otherwise the history looks like we changed the test -- and a full word is preferred over a 3-letter abbreviation usually.  But it's possible I am missing something here too...
Same comment for most of the changes up to line 300 of the test file.